source/_components/pioneer.markdown

```diff
@@ -19,6 +19,10 @@ To add a Pioneer receiver to your installation, add the following to your `confi
 media_player:
   - platform: pioneer
     host: 192.168.0.10
+    sources:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Thanks for the feedback - it totally makes sense. I restored the minimal configuration sample in 7c6d8b301d.

Reply:
👍